distribution/pom.xml

```diff
@@ -389,8 +391,8 @@
             <configuration>
                <outputDirectory>${project.build.directory}/distribution</outputDirectory>
                <workDirectory>${project.build.directory}/assembly/work</workDirectory>
-               <!-- Do not attach ZIP files to the Maven repository -->
-               <attach>false</attach>
+               <!-- (Do not attach ZIP files to the Maven repository) -->
+               <attach>true</attach><!-- we need that actually -->
```

Comment:
Please let us make this controllable via a property. For community we do NOT want to deploy distribution zips to Maven.

Reply:
I guess this can be removed from community too:

https://github.com/infinispan/infinispan/pull/6228/files#diff-9cf918424f312edfa0b7dd14b008eac5R174

Reply:
Indeed. Remember to rebase as well